boards/arm/degu_evk/degu_evk.dts

```diff
@@ -94,15 +94,15 @@
 &i2c0 {
 	compatible = "nordic,nrf-twim";
 	status = "okay";
-	sda-pin = <20>;
-	scl-pin = <22>;
+	sda-gpios = <&gpio0 20 0>;
+	scl-gpios = <&gpio0 22 0>;
 };
 
 &i2c1 {
 	compatible = "nordic,nrf-twim";
 	status = "okay";
-	sda-pin = <41>;
-	scl-pin = <11>;
+	sda-gpios = <&gpio1 9 0>;
```

Comment:
Why 9? Is it correct?

Reply:
41 / 32 = 1 (port)
41 % 32 = 9 (pin)

Seems correct to me.